### ISSUE TITLE
Hugo 0.58.0 breaks RSS

### ISSUE THREAD

**Issue by tastytea** [spookey/slick#7]
The people from hugo-academic have a fix and it works for slick too. I don't really understand what they are doing, that's why I didn't make a pull-request. :smile: 

https://github.com/gcushen/hugo-academic/commit/904fd887143b1a6f4cdcf5e7a2d45b4bf600ba19

**Comment by spookey** [comment]
Hey, thanks for the notice!

Currently I am working on some parts inside the ``develop`` branch. Yesterday I noticed the same behavior.
So I took the implementation to search for pages from the [default rss template](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/_default/rss.xml).
It ended up in this commit: 495e2fdf11d168e99e54ec897bcd95649bc953d8

If you want to test that in your setup, it would be nice if you check out the latest version of the ``develop`` branch. Otherwise the (now new) JavaScript file is missing.

It would be nice if you'll send me some update if it works with the ``develop`` branch.
I will keep this issue open, as long ``develop`` is not fully merged into ``master``.
